cdap-ui/app/cdap/components/Lab/experiment-list.tsx

```diff
@@ -35,6 +35,6 @@ export default [
       'Demo for new SchemaEditor. Includes complete rewrite in React + perf improvements',
```

Comment:
nit: perhaps remove the `Demo for` part from this description since this lab is not for the demo page anymore?